HW1_task1.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+ 
+int main() {
+  /*
+  HOMEWORK
+ ax^2 + bx + c = 0
+ */
+
+int a, b, c, discriminant, x1, x2; 
+cout<<endl<<"Enter values: "<<endl<<"a = ";
+cin>>a;
+cout<<"b = ";
+cin>>b;
+cout<<"c = ";
+cin>>c;
+
+
+discriminant = (b*b)-(4*a*c);
+
+
+if(discriminant<0) {cout<<"No roots of the equation";}
```

Comment:
тут стоит добавить return, так как дальше будет идти выполнения кода что есть не правильно
```suggestion
if ( discriminant<0 ) {
  cout<<"No roots of the equation";
  return;
}
```